guide/src/class.md

```diff
@@ -169,6 +169,23 @@ impl MyClass {
 }
 ```
 
+Alternatively, if your `new` method might throw errors you can return `PyResult<Self>`.
```

Comment:
To be consistent with the below table, maybe word this as "may fail" instead of "might throw errors"?

Reply:
Done